test/cmd.js

```diff
@@ -335,35 +381,6 @@ describe('express(1)', function () {
       .expect(404, /<h1>Not Found<\/h1>/, done);
     });
   });
-
-  describe('--help', function () {
```

Comment:
Sorry, wasn't aware of that. -h comes before --help yes. 

-h was the last test in the file , so they weren't in alphabetic order.